ingester/ingester.py

```diff
@@ -579,6 +635,10 @@ def main():
     VERBOSE = args.verbose
 
     logger.info("Starting ingestion process...")
+    start_http_server(int(os.environ.get("PROMETHEUS_PORT", 8002)))
```

Comment:
I don't know enough about how this is deployed, but I'm a bit suspicious on how you didn't need to change any compose file to enable this port